onnxruntime/core/graph/function_utils.h

```diff
@@ -14,13 +14,16 @@
 namespace onnxruntime {
 namespace function_utils {
 
-/** Create a OpSchema given a subgraph EP whant to fuse.
-* This is used when EP return fusion in GetCapability implementation.
-* @param graph The graph which host the subgraph.
-* @param nodes_to_fuse The metadata for the subgraph that EP want to fuse.
-*/
+/** Create a OpSchema given a subgraph EP want to fuse.
+ * This is used when EP return fusion in GetCapability implementation.
+ * @param graph The graph which host the subgraph.
+ * @param nodes_to_fuse The metadata for the subgraph that EP want to fuse.
+ * @param allow_anytype_tensor if true, we will set a ANY datatype as constraint,
+ * it means we allow every tensor types //all_tensor_types_with_bfloat.
+ */
```

Comment:
The 'ANY' concept is not something a new developer reading this documentation would have any idea about. 
May be clearer to say it will use a type constraint called TAggregatedTypes for all inputs and outputs, and that this will match all tensor types in the all_tensor_types_with_bfloat list.